server/blob/blob-deduplicating/doc/gc-properties.adoc

```diff
@@ -0,0 +1,23 @@
+= GC properties
+
+1. the execution time of the GC should be linked to
+active dataset but not to global dataset
+(for scalability purpose)
```

Comment:
What does that means? I understand "don't load the full dataset in memory at once.